system-test/mdbci/templates/default.json.template

```diff
@@ -86,13 +86,8 @@
         "name": "${product}",
         "version": "${version}",
         "force_version": ${force_backend_version}
-      },
-      {
-        "name": "galera_config",
-        "cnf_template" : "galera_server1.cnf"
       }
-    ],
-    "cnf_template_path": "${cnf_path}"
+    ]
```

Comment:
Why do the normal MariaDB nodes require the `cnf_template` arguments but the galera ones do not? Shouldn't they all be removed if they're copied over by the test code?